server/app/services/question/types/QuestionDefinition.java

```diff
@@ -237,6 +237,10 @@ public final ImmutableSet<CiviFormError> validate() {
     if (name.isBlank()) {
       errors.add(CiviFormError.of("Administrative identifier cannot be blank"));
     }
+    // Question names should not contain characters we can't handle in question paths.
+    if (!name.matches("^[a-zA-Z ]*$")) {
```

Comment:
Can you put the regex string in a constant called VALID_QUESTION_NAME_CHARACTERS or something of that ilk?